includes/admin/OrderSettings.php

```diff
@@ -296,6 +310,43 @@ private function setAllData(): void
         $this->setPickupLocation();
     }
 
+    private function setShipmentOptions(): void
+    {
+        $this->largeFormat     = $this->determineShipmentOption(
+            [$this, 'determineLargeFormat'],
+            '', // already taken into account by 'determineLargeFormat' callback
+            AbstractConsignment::SHIPMENT_OPTION_LARGE_FORMAT,
+            true
+        );
+        $this->returnShipment  = $this->determineShipmentOption(
+            [$this->shipmentOptions, 'isReturn'],
+            WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_RETURN,
+            AbstractConsignment::SHIPMENT_OPTION_RETURN
+        );
+        $this->onlyRecipient   = $this->determineShipmentOption(
+            [$this->shipmentOptions, 'hasOnlyRecipient'],
+            WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_ONLY_RECIPIENT,
+            AbstractConsignment::SHIPMENT_OPTION_ONLY_RECIPIENT
+        );
+        $this->signature       = $this->determineShipmentOption(
+            [$this->shipmentOptions, 'hasSignature'],
+            WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_SIGNATURE,
+            AbstractConsignment::SHIPMENT_OPTION_SIGNATURE
+        );
+        $this->sameDayDelivery = $this->determineShipmentOption(
```

Comment:
Ik vind dit brok wel eng, sommige dingen krijgen ineens true in die parameter en het is een grote foutgevoelige sliert voor mijn gevoel. In zo'n geval zie ik liever een cool loopje waarin je dit doet:

```php
$this->{$property} = $this->determineShipmentOption(
	  [$this->shipmentOptions, $item['shipment_options_method']],
	  $item['setting'],
	  $item['consignment_option'],
	  $item['wat_dit_doet'] ?? false
);
```

Dan hoef je alleen nog goed op te letten dat je array met die spulletjes erin wel klopt en hoef je niet 6x hetzelfde blok code te lezen om er bijvoorbeeld achter te komen of iets wel of niet `wat_dit_doet` gebruikt